_posts/en/newsletters/2020-07-22-newsletter.md

```diff
@@ -127,10 +127,10 @@ release candidates.*
   sending of [multipath payments][topic multipath payments]---payments
   which are split into several parts, with each part routed using a
   different path.  In brief, the algorithm C-Lightning uses splits
-  a payment into parts approximately 0.0001 BTC in value (each part having its
+  a payment into parts of approximately 0.0001 BTC in value (each part having its
   amount randomly fuzzed by plus or minus 10%).  If any sent part fails,
   that part is split into two parts (roughly in half, plus or minus 10%)
-  and the two parts are resent.  The PR additionally adds a
+  and the two parts are re-sent.  The PR additionally adds a
```

Comment:
I resentfully see that according to git grep, the homopath "resent" is used more often than "re-sent" in the writing, so happy to resentfully remove it :)